frontend/controller/actions/group.js

```diff
@@ -561,6 +563,32 @@ export default (sbp('sbp/selectors/register', {
       sbp('okTurtles.events/emit', REPLACE_MODAL, 'IncomeDetails')
     }
   },
+  'gi.actions/group/checkGroupSizeAndProposeMember': async function () {
+    // if current size of the group is >= 150, display a warning prompt first before presenting the user with
+    // 'AddMembers' proposal modal.
+
+    const { groupMembersCount } = sbp('state/vuex/getters')
+    const isGroupSizeLarge = groupMembersCount >= MAX_GROUP_MEMBER_COUNT
+
+    if (isGroupSizeLarge) {
+      const promptOpts = {
+        heading: 'Large group size',
+        question: L(
+          "Groups over 150 members are at significant risk for fraud, {a_}because it is difficult to verify everyone's identity.{_a} Are you sure that you want to add more members?",
+          { a_: `<a class='link' href='${ALLOWED_URLS.WIKIPEDIA_DUMBARS_NUMBER}'' target='_blank'>`, _a: '</a>' }
+        ),
+        yesButton: L('Yes'),
+        noButton: L('Cancel')
+      }
+
+      const yesButtonSelected = await sbp('gi.ui/prompt', promptOpts)
+      if (yesButtonSelected) {
+        sbp('okTurtles.events/emit', REPLACE_MODAL, 'AddMembers')
+      } else return false
```

Comment:
@SebinSong So, @dotmacro and I are not in complete agreement about how we should handle this scenario. She thinks we should simply prevent users from creating groups larger than 150, rather than allowing them the option to, since a lot of our messaging (and future blog posts) are all about having groups be a maximum of 150 members. I think she has a decent point, but I am not 100% convinced. However, because she feels more strongly on this issue than I do, I am willing to go along with her suggestion that we keep it locked down to 150 members.

However, I do not want to completely remove this code that you added. Instead, can you add a hard-coded boolean variable to the code, something like `const enforceDunbar = true`? If `enforceDunbar` is `true` (and by default it is), bring up a prompt that only has 1 button that says "OK", and do not allow users to create a proposal. If `enforceDunbar` is `false`, do what this PR is currently doing.

Sound OK?

Reply:
@taoeffect yes, that sounds good.